c_binding/libata.h

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (C) 2016 Red Hat, Inc.
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
```

Comment:
@joehandzik Sorry for the delay on answering this.  Based on this: https://www.gnu.org/licenses/gpl-faq.html#AllCompatibility  I believe we should keep everything in the library as LGPL 2.1.  The reason I say this is LGPL 3 is not compatible with clients that are GPL2 only.  Furthermore, the nvme library that @cathay4t is working on my be an issue too as having a LGPL 2.1 library using a LGPL 3 library creates the same license restrictions.